_posts/2018-02-26-conduit-0.3.1

```diff
@@ -0,0 +1,19 @@
+---
+title: Conduit 0.3.1 Released
+---
+
+Version 0.3.1 of Conduit is now available at: http://llnl-conduit.readthedocs.io/en/latest/releases.html
```

Comment:
@cyrush -- Would this link be better as: https://github.com/LLNL/conduit/releases ?

Reply:
@IanLee1521 I like to point folks at the docs, b/c it lands them in a tree with the rest of the docs (so at least they implicitly know the docs exist :-) )

Reply:
Sure, just the wording "Conduit is now available at:" which made me think that it could be simpler to have that be where they actually get the release. 

I'm totally on board with the link to that page staying at the bottom, right now you just link there to the same page twice. ;)